client/src/app/controllers/socket/index.coffee

```diff
@@ -0,0 +1,14 @@
+activity = require("../activity/activity.collection")
+
+class Socket
+  constructor: -> @initialize()
+
+  initialize: ->
+    # Connect to socket server
+    @socket = io("http://localhost:3000")
```

Comment:
This is to bypass the proxy. I set up a separate ticket to fix the proxy to support sockets